### ISSUE TITLE
Should we move all the message text to its own JSON?

### ISSUE THREAD

**Issue by vcidst** [mmcmd/verification-bot#19]
For the purpose of maintainability, you'd often be making changes to the phrasing of messages currently posted by the bot. Can we move the strings into its own JSON file?

They'll still be formatted to include variables so they'll need to be saved like this.

```
{
    "on_message": {
        "verify_begin": "Sorry, your account must be {0} or older in order to get verified. Please either verify with your phone number or contact the mod team through <@{1}}> (Moderator mail, top of the member list) for manual verification if you're unable to verify with a phone number.",
        "verify_done": "You have been verified. Please remember to read the <#{0}>. You may also assign yourself roles in <#{1}>. If you ever have an issue, do not hesitate to message <@{2}> (Moderator Mail, top of the member list)",
        "verify_abort": "You're already able to speak in the server, there is no need for you to get verified. Contact <@{0}> (Moderator Mail, top of the member list) if there is an issue."
    }
}
```

**Comment by mmcmd** [comment]
Yup, this sounds like an excellent idea

**Comment by mmcmd** [comment]
Included in #17
